.github/workflows/build-unified.yml

```diff
@@ -19,6 +19,9 @@ on:
         description: "Server branch name to deploy (examples: 'main', 'rc', 'feature/sm')"
         type: string
         default: main
+      is_workflow_call:
+        type: boolean
+        default: false
```

Comment:
Instead of the new input, the workflow trigger can be checked as follows: 
```
"$GITHUB_EVENT_NAME" == "workflow_dispatch"
```

```
if: ${{ github.event_name == "workflow_dispatch" }}
```

Reply:
the change on #325 actively use the input `is_workflow_call`